void tomo_db0::IoLoop::run_loop<false, true, false, true, false>()
--- PRE

+++ POST

@@ -530,7 +530,7 @@

 0a1d [ 3] nopl (%rax)
 0a20 [ 5] mov $0x1,%esi
 0a25 [ 3] mov %rbx,%rdi
-0a28 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0a28 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0a2d [ 5] jmp <+0x31b>
 0a32 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0a3d [ 3] nopl (%rax)
